integration/spark2/src/main/scala/org/apache/carbondata/spark/rdd/CarbonDataRDDFactory.scala

```diff
@@ -473,6 +473,13 @@ object CarbonDataRDDFactory {
             .STATUS_FILE_UPDATION_FAILURE
           return null
         }
+        // code to handle Pre-Priming cache for update command
```

Comment:
may be you can remove the empty if block above this

Reply:
done